webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -10,13 +10,15 @@
           />
         </hc-teaser-image>
         <ds-input model="title" class="post-title" placeholder="Title" name="title" autofocus />
+        <small style="width:100%;position:relative;left:90%">{{ form.title.length }}/64</small>
```

Comment:
I prefer to avoid inline styling @ogerly, not sure how others fell about it

Reply:
Would be nice(!) to make a class out of it … @ogerly 
But not necessarily …
Inline styling has the positive side of better analysability, if the same style is not used elsewhere.
Classes (can) have descriptive names telling us what this is about.